lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -90,9 +90,25 @@
         - success
     type: string
     sample: /etc/ssl/public/ansible.com.pem
+fingerprint:
+    description: The fingerprint of the public key. Fingerprint will be generated for each hashlib.algorithms available.
+    returned:
+        - changed
+        - success
+    type: hash
+    sample:
+    sample:
+      md5: "84:75:71:72:8d:04:b5:6c:4d:37:6d:66:83:f5:4c:29"
+      sha1: "51:cc:7c:68:5d:eb:41:43:88:7e:1a:ae:c7:f8:24:72:ee:71:f6:10"
+      sha224: "b1:19:a6:6c:14:ac:33:1d:ed:18:50:d3:06:5c:b2:32:91:f1:f1:52:8c:cb:d5:75:e9:f5:9b:46"
+      sha256: "41:ab:c7:cb:d5:5f:30:60:46:99:ac:d4:00:70:cf:a1:76:4f:24:5d:10:24:57:5d:51:6e:09:97:df:2f:de:c7"
+      sha384: "85:39:50:4e:de:d9:19:33:40:70:ae:10:ab:59:24:19:51:c3:a2:e4:0b:1c:b1:6e:dd:b3:0c:d9:9e:6a:46:af:da:18:f8:ef:ae:2e:c0:9a:75:2c:9b:b3:0f:3a:5f:3d"
+      sha512: "fd:ed:5e:39:48:5f:9f:fe:7f:25:06:3f:79:08:cd:ee:a5:e7:b3:3d:13:82:87:1f:84:e1:f5:c7:28:77:53:94:86:56:38:69:f0:d9:35:22:01:f4:7f:a8:69:ef:d9:8f:
+               dd:30:90:89:1e:a6:6a:60:97:82:fc:bf:c7:94:0f:9b"
 '''
 
 from ansible.module_utils.basic import *
```

Comment:
I know this isn't your code, though if you could update to only import what you need, rather than `*` that would be great
http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#python-imports